.circleci/scripts/auto-rebase.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# init git
+git config user.email "circleci@bolt.com"
+git config user.name "Circle CI"
+
+baseBranch="master"
+# add the branches that you want to auto-rebase
+# use space to sparate them (don't use comma)
+autoRebaseBranches=("jz/rebase-custom1" "jz/rebase-custom2")
```

Comment:
Could we have this list in a separate file, just to simplify its updating?

Reply:
fixed

Reply:
fixed, now it uses **_auto-rebase-branches.txt_** file to specify auto-rebasing branches.